libnavigation-ui/src/main/java/com/mapbox/navigation/ui/internal/route/RouteConstants.java

```diff
@@ -50,6 +50,6 @@
   public static final String MAPBOX_LOCATION_ID = "mapbox-location";
   public static final double MINIMUM_ROUTE_LINE_OFFSET = .001;
   public static final long ROUTE_LINE_VANISH_ANIMATION_DURATION = 1000;
-  public static final long ROUTE_LINE_VANISH_ANIMATION_DELAY = 300;
+  public static final long ROUTE_LINE_VANISH_ANIMATION_DELAY = 350;
```

Comment:
Is this needed after bumping to `9.3.0-beta.1`? It seems unrelated to OP

Reply:
It's unrelated but I found it to be a better value when testing for the crash. It didn't seem like it would do any harm including it here.